### PR TITLE
[ko] Update outdated files in dev-1.27-ko.1 (M198-M206)

### PR DESCRIPTION
**Pull request by jongwooo**
Signed-off-by: jongwooo <jongwooo.han@gmail.com>

## Description

Related Issue: #41631

Updated outdated files in `content/en/docs/tasks/run-application/`.

**9 files to be modified**  

- [x] M198. `content/en/docs/tasks/run-application/_index.md` | 1(+XS) 1(-)
- [x] M199. `content/en/docs/tasks/run-application/access-api-from-pod.md` | 24(+S) 15(-)
- [x] M200. `content/en/docs/tasks/run-application/delete-stateful-set.md` | 25(+S) 21(-)
- [x] M201. `content/en/docs/tasks/run-application/force-delete-stateful-set-pod.md` | 41(+M) 22(-)
- [x] M202. `content/en/docs/tasks/run-application/horizontal-pod-autoscale.md` | 38(+M) 41(-)
- [x] M203. `content/en/docs/tasks/run-application/run-replicated-stateful-application.md` | 15(+S) 30(-)
- [x] M204. `content/en/docs/tasks/run-application/run-single-instance-stateful-application.md` | 90(+M) 93(-)
- [x] M205. `content/en/docs/tasks/run-application/run-stateless-application-deployment.md` | 78(+M) 74(-)
- [x] M206. `content/en/docs/tasks/run-application/scale-stateful-set.md` | 16(+S) 17(-)

### CODE REVIEW

File: content/ko/docs/tasks/run-application/run-replicated-stateful-application.md
Comment (jongwooo) [inline content/ko/docs/tasks/run-application/run-replicated-stateful-application.md:1]: [edc8a4d](https://github.com/kubernetes/website/pull/47517/commits/edc8a4d3e8ed42c5b09af7adbedaf752070508a2)에서 CRLF에서 LF로 변경했습니다. 이후 변경사항은 [62b2982](https://github.com/kubernetes/website/pull/47517/commits/62b29820e40e5385b934c86dedb49b09d56fb2f8)에서 확인하실 수 있습니다.

File: content/ko/docs/tasks/run-application/delete-stateful-set.md [hunk 1]
```diff
@@ -76,15 +80,17 @@ kubectl delete pvc -l app.kubernetes.io/name=MyApp
...
-위의 예에서 파드에는 `app.kubernetes.io/name=MyApp` 라는 레이블이 있다. 사용자에게 적절한 레이블로 대체하자.
+위의 예에서 파드에는 `app.kubernetes.io/name=MyApp` 라는 레이블이 있다.
+사용자에게 적절한 레이블로 대체하자.
 
 ### 스테이트풀셋 파드의 강제 삭제
 
-스테이트풀셋의 일부 파드가 오랫동안 'Terminating' 또는 'Unknown' 상태에 있는 경우, apiserver에 수동적으로 개입하여 파드를 강제 삭제할 수도 있다. 이것은 잠재적으로 위험한 작업이다. 자세한 설명은 [스테이트풀셋 파드 강제 삭제하기](/ko/docs/tasks/run-application/force-delete-stateful-set-pod/)를 참고한다.
-
-
+스테이트풀셋의 일부 파드가 오랫동안'Terminating' 또는'Unknown' 상태에 있는 경우,
```
[the remark this answers is not in the document]
Comment (jongwooo) [reply content/ko/docs/tasks/run-application/delete-stateful-set.md]: @yoonian 리뷰 감사합니다! 해당 내용 수정하였습니다:)

**Comment by seokho-son** [comment]
Thanks!

/lgtm

**Pull request merged**